README.md

```diff
@@ -347,6 +347,20 @@ To avoid storage charges for retaining the temporary composite objects, consider
 ### Google Cloud Storage Class
 Due to the quick creation and deletion of composite objects, we recommend that only the [Standard storage class](https://cloud.google.com/storage/docs/storage-classes) is applied to your bucket to minimize cost and maximize performance.
 
+### Throughput and QPS Limits
+Many machine learning efforts opt for a highly distributed training model leveraging tools such as Pytorch Lightning and Ray. These models are compatible with Dataflux, but can often trigger the rate limits of Google Cloud Storage. This typically manifest in a 429 error, or slower than expected speeds while running distributed operations. Details on specific quotas and limits can be found [here](https://cloud.google.com/storage/quotas).
+
+#### Egress Throughput Limits
+429 errors acompanied with messages indicating `This workload is drawing too much egress bandwidth from Google Cloud Storage` or `triggered the Cloud Storage Egress Bandwidth Cap` indicate that the data throughput rate of your workload is exceeding the maximum capacity of your Google Cloud Project. To address these issues, you can take the following steps:
+
+1. Ensure that object composition is [disabled](#composite-objects) for dataflux download operations.  
+2. Check that other workloads executing within your project are not drawing excess bandwidth.
+3. Lower the scale of the workload in question to reduce its egress bandwidth (e.g. fewer running pods).
+4. Contact your Technical Account Manager or Google representative to file a Cloud Capacity Advisor request on your behalf. This request should specify that you wish to increase the bandwidth caps on your project.
```

Comment:
There are no results when I search ['gcp "Cloud Capacity Advisor"'](https://www.google.com/search?q=gcp+%22cloud+capacity+advisor%22&sca_esv=1fac4576b0bfa454&sxsrf=ADLYWIJ2mUa4QImCbFmvezANRtkuyzJD5Q%3A1720729464125&ei=eD-QZqCmB42B0PEP5-q_6AM&ved=0ahUKEwigwtrj6J-HAxWNADQIHWf1Dz0Q4dUDCA8&uact=5&oq=gcp+%22cloud+capacity+advisor%22&gs_lp=Egxnd3Mtd2l6LXNlcnAiHGdjcCAiY2xvdWQgY2FwYWNpdHkgYWR2aXNvciIyCBAhGKABGMMESKAFUNIBWJAEcAJ4AJABAJgBV6AB8QGqAQEzuAEDyAEA-AEBmAICoAKjAZgDAIgGAZIHATKgB8wI&sclient=gws-wiz-serp) on Google. Is there some better way we can refer to this or any existing documentation we can link?

Reply:
I pulled this name directly from the error message, so it could be a legacy term. I'll remove it.

Reply:
Oh, it could be ok then, but to avoid confusion you might just say something like "capacity request". In the second sentence, it would probably be good to specify "GCS bandwidth cap" also.